### PR TITLE
[PE-4742] Fixed broken Dropdown data-target

### PR DESCRIPTION
**Pull request by janibolkvadze**
This is a fix for a console error `this._popper.update() not a a function`

@dani-cl-madrid 

https://ctl.atlassian.net/browse/PE-4742

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://9a313f4c1441e391080887059648bce6072152a9-chi-pr.enterprise.ctl.io" target="_blank"> https://9a313f4c1441e391080887059648bce6072152a9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/888/ for more details.

File: src/chi/javascript/components/dropdown.js [hunk 1]
```diff
@@ -235,9 +235,6 @@ class Dropdown extends Component {
       Util.addClass(this._elem, CLASS_ACTIVE);
       Util.addClass(this._elem, CLASS_HAS_ACTIVE);
       Util.addClass(this._dropdownElem, CLASS_ACTIVE);
-      if (this._popper) {
```
Comment (dani-cl-madrid) [inline src/chi/javascript/components/dropdown.js:238]: Popper.update must be called to re-position the popover before being shown. If it fails in the console from time to time, please make a better check for the function existance, i.e.:
```Javascript
if (this._popper && typeof this._popper.update === "function") {
  this._popper.update();
}
```

Comment (janibolkvadze) [reply src/chi/javascript/components/dropdown.js:238]: I see, thanks!

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://bdda08de2c4b6507ea61c8c6310a0a8358f26b0d-chi-pr.enterprise.ctl.io" target="_blank"> https://bdda08de2c4b6507ea61c8c6310a0a8358f26b0d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-52ei18, https://ops.elasticbox.com/services/instances/i-v53up3

**Pull request merged**